library/build.gradle

```diff
@@ -105,6 +105,10 @@ task integrationTest(type: Test) {
     // Force tests to always be re-run, since integration tests involve communicating with external
     // resources.
     outputs.upToDateWhen { false }
+
+    testLogging {
+        events = ["passed", "failed", "skipped", "started", "standard_out", "standard_error"]
```

Comment:
I found this useful while monitoring GHA runs of ITs, rather than 30ish minutes of no output.